nodeup/pkg/model/docker.go

```diff
@@ -850,15 +589,6 @@ var dockerVersions = []packageVersion{
 		},
 	},
 
-	// 19.03.7 - Linux Generic
-	{
-		PackageVersion: "19.03.7",
-		PlainBinary:    true,
-		Architectures:  []Architecture{ArchitectureAmd64},
-		Source:         "https://download.docker.com/linux/static/stable/x86_64/docker-19.03.7.tgz",
-		Hash:           "5f7199aa237cc8fa10b95ee0c06c5e9ca9ad4296",
-	},
-
```

Comment:
Unintended removal?

Reply:
This is an intermediary release that was added just to test the new .tgz support. It was never released so there is no need to keep it around.

Reply:
I would have expected it to be in a separate commit then. It is a separate concern and later archaeology might benefit from the separate commit message.

Reply:
Seems reasonable, will move it to a separate commit.